src/ILLink.Shared/TypeSystemProxy/IMemberProxy.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace ILLink.Shared.TypeSystemProxy
+{
+	interface IMemberProxy
```

Comment:
You could also use a struct here with implicit conversions from methods

Reply:
Yes - I'll leave it as is for now - and see how it goes. Currently it's actually not needed for anything, but it will be soon.